conftest.py

```diff
@@ -176,37 +168,61 @@ def function_to_seed_torch(seed):
 
 
 @pytest.fixture(autouse=True)
-def autoseeding_of_everything(record_property, request):
+def autoseeding_of_everything(request):
     """Seed everything we can, for determinism."""
-    main_seed = request.config.getoption("--forcing_random_seed", default=None)
+    randomly_seed = request.config.getoption("--randomly-seed", default=None)
 
-    if main_seed is None:
-        main_seed = random.randint(0, 2**64 - 1)
+    if randomly_seed is None:
+        raise ValueError("why isn't --randomly-seed set?")
 
-    seed = main_seed
-    record_property("main seed", main_seed)
+    # We need to find the relative file path of the test file. It does not look native with request,
+    # so we recompute it.
+    absolute_path = str(request.fspath)
+
+    assert (
+        absolute_path.count("concrete-ml/test") == 1
```

Comment:
not sure what this is, and the associated error remains pretty vague !

Reply:
I have an absolute path, eg `/Users/benoitchevalliermames/Documents/Zama/Git/concrete-ml/tests/common/test_utils.py`
I want to compute `tests/common/test_utils.py` from it
It will work, as soon as the guy making the cloning didn't set a crazy path like

`/something/concrete-ml/tests/something-else/concrete-ml/tests/`

in which case I want to abort because it's not good.

Reply:
If you really want, I can make a change to find the _last_ occurence of `concrete-ml/tests`, instead of the first one. But except if the guys want to bother us, never the assert will fail

Reply:
ok

Reply:
done, ok to resolve?

Reply:
yes thanks !